### PR TITLE
Add more examples for the file source plugins

### PR DESCRIPTION
**Pull request by bgruening**
## What did you do? 

I added more examples to the file_sources configuration. Real-world examples are also included to serve as documentation but also spread their adoption.

## Why did you make this change?

Better documentation.

## How to test the changes? 

- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [x] Instructions for manual testing are as follows:

  Start Galaxy with this configuration and use the new data sources.

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Comment by bgruening** [comment]
ping @hexylena and @bernt-matthias

File: lib/galaxy/config/sample/file_sources_conf.yml.sample [hunk 1]
```diff
@@ -14,23 +14,142 @@
...
+  doc: European Commission’s Copernicus Earth Observation Programme. Sentinel-3 is a polar orbiting satellite that completes 14 orbits of the Earth a day.
+  bucket: meeo-s3/NRT/
+  anon: true
 
 - type: s3fs
-  label: My MinIO storage
-  endpoint_url: "https://minio.usegalaxy.eu"
-  id: galaxy-minio-storage
-  doc: Galaxy MinIO S3 storage
-  anon: false
```
Comment (hexylena) [inline lib/galaxy/config/sample/file_sources_conf.yml.sample]: Let's keep this one too?

**Review by bernt-matthias — APPROVED** [review]
Maybe add some link to the docs how to configure the user preferences for `webdav` and dropbox?

**Review by jmchilton — APPROVED** [review]

**Comment by jmchilton** [comment]
Added documentation about user preferences makes a lot of sense, but it can be in a follow up pull request. Thanks a ton @bgruening

**Pull request merged**